Foundation/HTTPCookieStorage.swift

```diff
@@ -103,10 +121,12 @@ open class HTTPCookieStorage: NSObject {
     */
     open class var shared: HTTPCookieStorage {
         get {
-            if sharedStorage == nil {
-                sharedStorage = HTTPCookieStorage(cookieStorageName: "shared")
+            return sharedSyncQ.sync {
```

Comment:
this looks like a dispatch_once style; shouldn't that be fine as a lazy var?

Reply:
- lazy vars aren't thread safe (https://bugs.swift.org/browse/SR-1042)
- there's no `dispatch_once` in Swift

Reply:
@weissi - How about open static var shared: HTTPCookieStorage?